libraries/AP_BattMonitor/AP_BattMonitor_DJI.cpp

```diff
@@ -0,0 +1,79 @@
+#include <AP_HAL/AP_HAL.h>
+#include "AP_BattMonitor_DJI.h"
+#include <GCS_MAVLink/GCS.h>
+#include <AP_SerialManager/AP_SerialManager.h>
+#include <AP_Math/crc.h>
+
+/*
+  "battery" monitor for DJI Battery via serial port.
+
+   For future Referance:
+   Total Voltage : 8,9
+   Percentage : 17
+   Cell1 :  20,21
+   Cell2 :  22,23
+   Cell3 :  24,25
+   Cell4 :  26,27
+   
+ */
+extern const AP_HAL::HAL& hal;
+
+ #define DELAY_TIME_US      1000000     // Battery info requesting delay
```

Comment:
A better name might be `REQUEST_INTERVAL_US`. We'd like the data at 10Hz if we can get it though, because we do use the voltage in motor control for example.

Reply:
name changed to REQUEST_INTERVAL_US , increasing rate needs more testing,